### PR TITLE
Added capture loading support to OrbitStartupWindow

### PR DESCRIPTION
**Pull request by beckerhe**
This PR adds a button to Orbit's startup window which allows to load a capture without connecting to a remote profile target.

image

### CODE REVIEW

**Comment by ronaldfw** [comment]
Not looked at code, but UI looks good to me.

File: OrbitQt/main.cpp [hunk 1]
```diff
@@ -154,7 +174,8 @@ static void StyleOrbit(QApplication& app) {
   darkPalette.setColor(QPalette::HighlightedText, Qt::black);
   app.setPalette(darkPalette);
   app.setStyleSheet(
-      "QToolTip { color: #ffffff; background-color: #2a82da; border: 1px solid "
+      "QToolTip { color: #ffffff; background-color: #2a82da; border: 1px "
+      "solid "
```
Comment (antonrohr) [inline OrbitQt/main.cpp:178]: nit: I think this got auto formatted when it was indented deeper.

File: OrbitQt/orbitmainwindow.cpp [hunk 2]
```diff
@@ -679,24 +679,31 @@ void OrbitMainWindow::on_actionSave_Capture_triggered() {
...
 void OrbitMainWindow::on_actionOpen_Capture_triggered() {
   QString file = QFileDialog::getOpenFileName(
-      this, "Open capture...", Path::GetCapturePath().c_str(), "*.orbit");
+      this, "Open capture...", QString::fromStdString(Path::GetCapturePath()),
+      "*.orbit");
   if (file.isEmpty()) {
     return;
   }
 
+  (void) OpenCapture(file.toStdString());
```
Comment (antonrohr) [inline OrbitQt/orbitmainwindow.cpp:688]: what happens when an error occurs and the result is discarded like that? Does it raise an exception?

Comment (beckerhe) [reply OrbitQt/orbitmainwindow.cpp:688]: No exception is raised. The (void) is just needed to make the nodiscard attribute happy.

File: OrbitQt/main.cpp (hunk 1 above)
Comment (beckerhe) [reply OrbitQt/main.cpp]: fixed. thanks.

File: OrbitQt/main.cpp [hunk 3]
```diff
@@ -61,45 +61,61 @@ using ServiceDeployManager = OrbitQt::ServiceDeployManager;
...
 static outcome::result<void> RunUiInstance(
     QApplication* app,
     std::optional<DeploymentConfiguration> deployment_configuration,
     ApplicationOptions options) {
-  const uint16_t asio_port = absl::GetFlag(FLAGS_asio_port);
-  const uint16_t grpc_port = absl::GetFlag(FLAGS_grpc_port);
   std::optional<OrbitQt::ServiceDeployManager> service_deploy_manager;
 
-  OUTCOME_TRY(ports, [&]() -> outcome::result<Ports> {
+  OUTCOME_TRY(result, [&]() -> outcome::result<std::tuple<Ports, QString>> {
```
[the remark this answers is not in the document]
Comment (beckerhe) [reply OrbitQt/main.cpp:93]: discussed OOB.

**Pull request merged**